src/playground/src/ui/screen/tabNavigator/tabScreen1.component.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {
+  View,
+  Text,
+} from 'react-native';
+
+export class TabScreen1 extends React.PureComponent {
```

Comment:
TabScreen components could be simplified to SFCs which would be used in your `TabNavigatorScreen`. So you will not need 3 files

Reply:
Nice catch! Will be refactored.